File: C/3342.c
#include <stdio.h>

int main(){

    int n, nBrancas = 0, nPretas;
    scanf("%i", &n);

    n = n*n;

    nPretas = n/2;

    if (n%2==0){
        nBrancas = nPretas;
    }else{
        nBrancas = (n+1) / 2;
    }

    printf("%i casas brancas e %i casas pretas\n", nBrancas, nPretas);

    return 0;
}

File: Python/3342.py
n = int(input())
n = n*n

nPretas = int(n/2)
nBrancas = 0

if n % 2 == 0:
    nBrancas = nPretas
else:
    nBrancas = int((n+1) / 2)

print(f"{nBrancas} casas brancas e {nPretas} casas pretas")